packages/time/src/index.ts

```diff
@@ -6,11 +6,13 @@ function pad(s: string | any[]) {
 }
 
 export function convertTimeToMins(seconds: number) {
+  if (seconds < 0) return '0m'
```

Comment:
I'd just change seconds to 0
```suggestion
  if (seconds < 0) seconds = 0
```